### PR TITLE
Add links

### PR DESCRIPTION
**Pull request by EliiseS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,18 @@ This is a hands-on workshop where you will develop, from scratch, a browser game
...
+To get started, view the prerequisites below, then see the intro module [here](modules/00_intro/README.md).
+
+Full list of all Sections / modules:
+
+- [⚒️ Intro](modules/00_intro/README.md)
+- [🚦 Hello world](modules/01_hello_world_api/README.md)
+- [📦 Templating](modules/02_templating/README.md)
+- [❇️ In memory storage](modules/03_in_memory_storage/README.md)
+- [🚀 Image upload](modules/04_add_images/README.md)
+- [🌐 Pull request](modules/10_pull_request_pipeline/)
```
Comment (LauraDamianTNA) [inline README.md:16]: Should we add the README file here as well?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,18 @@ This is a hands-on workshop where you will develop, from scratch, a browser game
...
+- [💻 Deploying to Azure](modules/11_deploying_to_azure/)
```
Comment (LauraDamianTNA) [inline README.md:17]: Same here, we can add the README to be consistent

File: README.md (hunk 1 above)
Comment (laura-damian) [inline README.md]: Just to be consistent
```suggestion
- [🌐 Pull request](modules/10_pull_request_pipeline/README.md)
```

File: README.md (hunk 2 above)
Comment (laura-damian) [inline README.md]: Just to be consistent
```suggestion
- [💻 Deploying to Azure](modules/11_deploying_to_azure/README.md)
```

**Review by laura-damian — APPROVED** [review]
Thanks, we need to remember to keep this up to date when we add more modules.

**Pull request merged**